packages/api/server/controllers/townController/findClosedNearbyTowns.js

```diff
@@ -0,0 +1,14 @@
+
+module.exports = models => async (req, res, next) => {
```

Comment:
Pour des raisons de testabiliité, il est préférable d'éviter l'injection de dépendance ici, et require directement la fonction que tu souhaites en haut du fichier.

`const { findNearby } = require('#server/models/shantytownModel');`

Je sais que ce n'est pas ce qu'on fait ailleurs, mais c'est une erreur "historique" qu'on essaie de corriger petit à petit. ;)

Reply:
bien noté ! Tant qu'à faire je vais corriger le fichier findNearbyTowns avec